src/renderer/commands/drawObject.ts

```diff
@@ -33,11 +34,21 @@ export interface DrawObjectProps {
 }
 
 /*
- * Shader to draw a single object with Phong shading to the screen
+ * Shader to draw a single object with Phong shading to the screen.
+ *
+ * @param {REGL.regl} regl: The regl object we're drawing.
```

Comment:
nit: `regl` is like a factory to build a function to draw an object rather than the object being drawn